playbook_cloudkit_create_hosted_cluster_post_install.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Cloudkit hosted cluster post-install configuration
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  environment:
+    KUBECONFIG: "{{ admin_kubeconfig | cloudkit.service.to_temp_file }}"
+
+  vars:
+    cluster_order: "{{ ansible_eda.event.payload }}"
+
+  pre_tasks:
+    - name: Extract template info
+      ansible.builtin.include_role:
+        name: extract_template_info
+
+  tasks:
+    - name: Post-install hosted cluster configuration
+      ansible.builtin.include_role:
+        name: "{{ template_id }}"
+        tasks_from: post_install.yaml
```

Comment:
For consistency, hould we update `playbook_cloudkit_create_hosted_cluster.yml` to explicitly use `install.yaml` rather than defaulting to `main.yaml`?

Reply:
On second thought, maybe not, since it performs both install and teardown activities based on parameters.

Reply:
For the sake of consistency, I was thinking about removing `state` parameter, and instead include `install.yml`, `post-install.yml` or `delete.yml` depending on the context.